css-view-transitions-1/Overview.bs

```diff
@@ -166,15 +455,6 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 			Note: The value <css>none</css> is invalid as a <<custom-ident>>.
 	</dl>
 
-	The [=document element=] participates in a view transition by default using the following rule,
-	added to the [=HTML user agent style sheet=]:
-
-	```css
-	html {
-		view-transition-name: root;
-	}
-	```
```

Comment:
I felt this was kinda hidden away, so I moved it into the UA styles section (which used to be the UA keyframes section)